grails-app/views/email/_shipmentItemReceived.gsp

```diff
@@ -8,7 +13,7 @@
             ${warehouse.message(code: 'email.user.label', args: [recipient?.firstName]) },
         </div>
         <div>
-            ${warehouse.message(code: 'email.shipmentItemReceived.label') }:
+            ${warehouse.message(code: 'email.shipmentItemReceived.label', args: [shipmentInstance?.destination]) }:
```

Comment:
Same thing here. We should use the name property rather than relying on the toString() method.